dashboard/src/actions/apps.ts

```diff
@@ -172,8 +172,7 @@ export function getAppWithUpdateInfo(
         app.chart &&
         app.chart.metadata &&
         app.chart.metadata.name &&
-        app.chart.metadata.version &&
-        app.chart.metadata.appVersion
+        app.chart.metadata.version
```

Comment:
It can be simplified using the `?.`  operator, I think (but fine as is)

```js
if(
 app?.chart?.metadata?.name &&
 app?.chart?.metadata?.version &&
 app?.chart?.metadata?.appVersion
){
...
```

Reply:
yes, that is a shorter way to write the condition. but here i was looking at only making changes for the bug fixes, refactoring was not really necessary.

for the other file, ?? is what i was looking for and is so much nicer to use (next time :-))